components/article/stylesheets/index.styl

```diff
@@ -90,13 +103,19 @@ body.body-article
     cursor pointer
     garamond-size('l-body')
     margin-left 10px
+    @media screen and (max-width 400px)
+      garamond-size('s-body')
   p
     margin-bottom 1em
     &:last-child
       margin-bottom 0
+    @media screen and (max-width 400px)
+      margin-top 1em
   h2
     font-size 28px
     line-height 1.2em
+    @media screen and (max-width 400px)
+      font-size 24px
```

Comment:
Oops were these the font sizes on MG? If so, we might want to adjust some of these to match instead to https://www.artsy.net/style-guide/typography as much as possible -- there's a `garamond-size('s-headline')` that's 25px that could match this one.